crm_lead_to_event_registration/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Regenerate from `oca-gen-addon-readme`

Reply:
oca-gen-addon-readme will be regenerate after merge.